endpoints.json

```diff
@@ -72,5 +72,9 @@
       "article_id": 2,
       "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
     }
+  },
+  "DELETE /api/comments/:comment_d": {
+    "description": "deletes comment of specific id and reponds with no content",
+    "exampleResponse": {}
```

Comment:
As you have described that no content is given there isnt a need for an example response to be given here as well